lib/config.go

```diff
@@ -182,6 +182,12 @@ type RuntimeConfig struct {
 
 	// BindMountPrefix is the prefix to use for the source of the bind mounts.
 	BindMountPrefix string `toml:"bind_mount_prefix"`
+
+	// SubUidName is the name to lookup in /etc/sub_uid for setting up the userNS.
+	SubUidName string `toml:"sub_uid_name"`
```

Comment:
Do we want to support UidMap in CRI-O or force their to be an entry in /etc/subuid?
@mrunalp WDYT?